app/code/Magento/Sales/Model/Order/CreditmemoFactory.php

```diff
@@ -41,15 +47,18 @@ class CreditmemoFactory
      *
      * @param \Magento\Sales\Model\Convert\OrderFactory $convertOrderFactory
      * @param \Magento\Tax\Model\Config $taxConfig
+     * @param FormatInterface $localeFormat
      * @param \Magento\Framework\Serialize\Serializer\Json $serializer
      */
     public function __construct(
         \Magento\Sales\Model\Convert\OrderFactory $convertOrderFactory,
         \Magento\Tax\Model\Config $taxConfig,
+        FormatInterface $localeFormat,
         \Magento\Framework\Serialize\Serializer\Json $serializer = null
     ) {
         $this->convertor = $convertOrderFactory->create();
         $this->taxConfig = $taxConfig;
+        $this->localeFormat = $localeFormat;
         $this->serializer = $serializer ?: \Magento\Framework\App\ObjectManager::getInstance()->get(
```

Comment:
Remove backward compatible class loader and add your dependency as last.

Reply:
The changes has been added, thank you.